conda-store-server/conda_store_server/build.py

```diff
@@ -298,14 +300,20 @@ def build_conda_docker(conda_store, build):
         output_image=f"{build.specification.name}:{build.build_key}",
         records=records,
         default_prefix=info["env_vars"]["CONDA_ROOT"],
+        pip_files=pip_files,
         download_dir=download_dir,
         user_conda=user_conda,
         channels_remap=info.get("channels_remap", []),
         layering_strategy="layered",
     )
 
+    conda_store.log.info(
+        f"Is {schema.BuildArtifactType.DOCKER_MANIFEST} in build_artifacts?"
+    )
+    conda_store.log.info(f"build_artifacts: {conda_store.build_artifacts}")
+
     if schema.BuildArtifactType.DOCKER_MANIFEST in conda_store.build_artifacts:
         conda_store.container_registry.store_image(conda_store, build, image)
 
     if schema.BuildArtifactType.CONTAINER_REGISTRY in conda_store.build_artifacts:
-        conda_store.container_registry.push_image(conda_store, build, image)
+        conda_store.container_registry.push_image(conda_store, build, image)
```

Comment:
It's a good idea to set VSCode's option to ensure newline at the end of files to avoid these types of changes popping up in commits
https://stackoverflow.com/a/44704969